mod_ood_proxy/lib/nginx.lua

```diff
@@ -54,7 +54,7 @@ function nginx_handler(r)
     if redir then
       local pun_app_request = redir:match("^" .. pun_uri .. "(/.+)$")
       if not pun_app_request then return http.http404(r, "bad `redir` request (" .. redir .. ")") end
-      return http.http302(r, pun_app_request)  -- ignore errors
+      return http.http302(r, redir)  -- ignore errors
```

Comment:
This works and it all looks good, I'm just trying to understand some Lua here. I'm surprised that using `redir` and not `pun_app_request` is what is needed and was curious why that is the case.

Reply:
`redir:match("^" .. pun_uri .. "(/.+)$")` is group matching, so what `pun_app_request` ends up being `/sys/dashboard/` without the `pun_uri`. (i.e,m should be `/pun/sys/dashboard`).